sklearn/preprocessing/imputation.py

```diff
@@ -107,6 +107,8 @@ class Imputer(BaseEstimator, TransformerMixin):
     statistics_ : array of shape (n_features,)
         The imputation fill value for each feature if axis == 0.
 
+    imputed_features : array of shape (n_features_with_missing, )
+        The input features which
```

Comment:
I think documenting this clearly will help make sure logic is correct. So please write what this should contain.